prow/scripts/cluster-integration/helpers/create-cluster.sh

```diff
@@ -21,41 +21,31 @@ function createCluster() {
 		CLUSTER_NAME
 		STANDARIZED_NAME
 		TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS
+		GCLOUD_NETWORK_NAME
+		GCLOUD_SUBNET_NAME
+		DNS_SUBDOMAIN DNS_DOMAIN
 	)
 
 	utils::check_required_vars "${requiredVars[@]}"
 
-	if [[ -z "${PERFORMACE_CLUSTER_SETUP}" ]]; then
-		requiredVars+=(
-			GCLOUD_NETWORK_NAME
-			GCLOUD_SUBNET_NAME
-			DNS_SUBDOMAIN DNS_DOMAIN
-		)
+	log::info "Reserve IP Address for Ingressgateway"
+	GATEWAY_IP_ADDRESS_NAME="${STANDARIZED_NAME}"
+	GATEWAY_IP_ADDRESS=$(gcloud::reserve_ip_address "${GATEWAY_IP_ADDRESS_NAME}")
+	echo "Created IP Address for Ingressgateway: ${GATEWAY_IP_ADDRESS}"
+
+	log::info "Create DNS Record for Ingressgateway IP"
+	GATEWAY_DNS_FULL_NAME="*.${DNS_SUBDOMAIN}.${DNS_DOMAIN}"
+	gcloud::create_dns_record "${GATEWAY_IP_ADDRESS}" "${GATEWAY_DNS_FULL_NAME}"
+
+	NETWORK_EXISTS=$("${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/network-exists.sh")
+	if [ "$NETWORK_EXISTS" -gt 0 ]; then
+		log::info "Create ${GCLOUD_NETWORK_NAME} network with ${GCLOUD_SUBNET_NAME} subnet"
+		"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-network-with-subnet.sh"
+	else
+		log::info "Network ${GCLOUD_NETWORK_NAME} exists"
 	fi
```

Comment:
This one can be changed to function `gcloud::create_network`. But looking at that hasn't this been changed already?

Reply:
This script was removed in merged PR #3132